app/.editorconfig

```diff
@@ -0,0 +1,16 @@
+root = true
```

Comment:
[.editorconfig](https://editorconfig.org/) is supported by some editors natively or through dedicated plugins.

Prettier, for example, uses the options defined in this file to initialize its own [options](https://prettier.io/docs/en/options.html) (when they are not overwritten by options from .prettierrc).

I'm not personally a big fan of Prettier, and I decided that the UI code didn't look like it is going to be changed often by many developers, and instead of _configuring_ Prettier I added this file - more as a way of suggesting the basic formatting rules than rigidly enforcing them.